README.md

```diff
@@ -85,6 +85,50 @@ To indicate your project is using `very_good_analysis` →
 [![style: very good analysis](https://img.shields.io/badge/style-very_good_analysis-B22C89.svg)](https://pub.dev/packages/very_good_analysis)
 ```
 
+## Linter rules
+
+### Enabled rules
+
+To see all the rules that are enabled, check the [analysis_options.yaml][analysis_options_yaml].
+
+### Excluded rules
```

Comment:
@tomarra shall we remove this section until we derive a reason for some of them? Or shall we instead keep it and update the reasons as we specify them in follow-up Pull Requests.